Kickstarter-iOS/DataSources/ProjectPamphletContentDataSource.swift

```diff
@@ -39,7 +39,7 @@ internal final class ProjectPamphletContentDataSource: ValueCellDataSource {
     }
 
     self.set(
-      values: [projectAndRefTag],
+      values: [(project, refTag)],
```

Comment:
We will update `ProjectPamphletMainCell` to be configured with all three values (or just `ProjectPamphletContentData`).